Library/Homebrew/patch.rb

```diff
@@ -64,9 +64,12 @@ def contents
   end
 
   def apply
+    Utils.ensure_git_installed!
+
     data = contents.gsub("HOMEBREW_PREFIX", HOMEBREW_PREFIX)
-    cmd = "/usr/bin/patch"
-    args = %W[-g 0 -f -#{strip}]
+    cmd = "/usr/bin/git"
```

Comment:
We actually can't guarantee that git is at `/usr/bin/git`; prior to Xcode 4, Apple didn't ship it.